RecoParticleFlow/Configuration/python/customizePF.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+#
+# customization function for ECAL PF-rechit thresholds
+# - ZS everywhere
+# - 1-sigma noise equivalent from 2018
+#
+def ecalNoSRPFCut1sigma(process):
+   import RecoParticleFlow.PFClusterProducer.particleFlowZeroSuppressionECAL_cff as _pfZS
+   if hasattr(process, "particleFlowRecHitECAL"):
+      process.particleFlowRecHitECAL.producers[0].srFlags = ""
+      process.particleFlowRecHitECAL.producers[1].srFlags = ""
+      process.particleFlowRecHitECAL.producers[0].qualityTests.thresholds = _pfZS._particle_flow_zero_suppression_ECAL_2018.thresholds
+      process.particleFlowRecHitECAL.producers[1].qualityTests.thresholds = _pfZS._particle_flow_zero_suppression_ECAL_2018.thresholds
+      process.particleFlowRecHitECAL.producers[0].qualityTests.applySelectionsToAllCrystals = cms.bool(True)
+      process.particleFlowRecHitECAL.producers[1].qualityTests.applySelectionsToAllCrystals = cms.bool(True)
```

Comment:
@amassiro 
did you confirm that this method works?

Looking more carefully, it seems like all the "qualityTests" should be changed to ```qualityTests[0]```